src/screens/main_screen.rs

```diff
@@ -81,6 +81,16 @@ impl DrawableComponent for MainScreenLayer {
                 app.shutdown();
                 EventResult::Consumed
             }
+            KeyCode::Char('s') => {
+                app.task_store.tasks.sort_by_key(|t| t.priority.clone());
```

Comment:
Tiny nit: `Priority` could derive `Copy` which would allow the removal of `.clone` from here and below.

Reply:
Oops, sorry I put the changes to add some `derive`s in a different commit accidentally. Give me a sec

Reply:
Done